### PR TITLE
feat: not return password value

### PR DESCRIPTION
**Pull request by victorggonzalez**

### CODE REVIEW

**Review by pyphilia — CHANGES_REQUESTED** [review]
Thanks for the PR! 
But actually, from what we decided for the password mechanism, we don't need the password field (even as a boolean). So I think the best solution to not return the password is to add a schema to this endpoint. Let me know if it's not clear^^

**Comment by victorggonzalez** [comment]
Done! Let me know what you think of this

File: src/services/members/schemas.ts [hunk 1]
```diff
@@ -35,6 +47,14 @@ export default {
   },
 };
 
+// schema for getting current member
+const getCurrent = {
+  // params: { $ref: 'http://graasp.org/#/definitions/idParam' },
```
Comment (pyphilia) [inline src/services/members/schemas.ts:52]: Remove.

File: src/services/members/schemas.ts [hunk 2]
```diff
@@ -13,6 +13,18 @@ export default {
...
     },
 
+    currentMember: {
+      type: 'object',
+      properties: {
+        id: { type: 'string' },
+        name: { type: 'string' },
+        email: { type: 'string' },
+        type: { type: 'string' },
+        extra: { type: 'object', additionalProperties: true },
```
Comment (pyphilia) [inline src/services/members/schemas.ts:23]: Maybe add createdAt and updatedAt?
Change to `additionalProperties: false`?

**Review by pyphilia — CHANGES_REQUESTED** [review]
Looks good! Do you think you could update the test for /members/current? (member.test.ts) We would like to test that password does not exist in the response, but the other fields do.

**Review by pyphilia — APPROVED** [review]

**Pull request merged**